client/src/components/Workouts/WorkoutDetail.jsx

```diff
@@ -0,0 +1,39 @@
+
+const WorkoutDetail = () => {
+    const workout = {
+        //able to add more as needed
+        title: "Sample Workout",
+        videoUrl: "https://www.youtube.com/watch?v=ErZhd3qvPYs",
+        instructions: "Go Brazy For five sets of six reps",
+        type: "Cardio",
+        muscle: "Legs",
+        difficulty: "Intermediate"
+    };
+
+    return (
+        <div className="bg-blue-100 py-8">
```

Comment:
Make it extend out the whole screen 
consider using the following styles `h-screen` and `w-screen` so that the blue extends the whole page